test/ui-test/extensionUITest.ts

```diff
@@ -130,6 +90,20 @@ export function extensionsUITest(): void {
       });
     });
 
+    if (!clusterIsAvailable) {
+      it('should notify user that he must log into a cluster', async function context() {
```

Comment:
If this is setup, then it does not need to be in an `it`. If it is a test, where is the `expect`?

Reply:
not a setup, it is a check for dialog that talks to user. but I will add expect, here it makes sense.

Reply:
ok, thanks